src/modules.js

```diff
@@ -160,4 +184,105 @@ if (__analytics.amplitude.isEnabled) {
 }
 
 
+// --- Infermedica Analytics ---
+if(__analytics.infermedicaAnalytics?.isEnabled) {
+  const infermedicaModule = function () {
+    const hasAuthentication = false;
+    const baseURL = __analytics.infermedicaAnalytics?.baseURL || 'https://analytics-proxy.test.infermedica.com/';
+    const {topic, environment, firebaseConfig} = __analytics.infermedicaAnalytics;
+    const browser = Bowser.getParser(window.navigator.userAgent);
+    const analyticsApi = axios.create({
+      baseURL,
+    });
+    const publish = async function(data) {
+      const publishURL = '/api/v1/publish';
+      const attributes = {
+        environment,
+      }
+      const events = [
+        {
+          data
+        }
+      ];
+      const payload = JSON.stringify({
+        topic,
+        events,
+      });
+
+      await analyticsApi.post(publishURL, payload);
```

Comment:
Q: Are you sure request is made correctly? Look here: https://masteringjs.io/tutorials/axios/post-json

Seams like axios by default serializes objects to JSON and sets proper Content-Type header. On the other hand if you give have a string (already serialised JSON) it does not set the Content-Type correctly neither do you.

Reply:
when I send payload without this changes I have error from this endpoint :D